app/javascript/pages/SignIn.tsx

```diff
@@ -3,9 +3,12 @@ import React from "react"
 import { SignInForm } from "../authentication/SignInForm"
 import FormsLayout from "../layouts/FormLayout"
 import withAppSetup from "../layouts/withAppSetup"
+interface SignInProps {
+  assetPaths: unknown
+}
 
-const SignIn = () => (
-  <FormsLayout>
+const SignIn = (props: SignInProps) => (
+  <FormsLayout assetPaths={props.assetPaths}>
```

Comment:
can we provide asset paths through context? Maybe we can pair on this because I have some big ideas on how we can do this